wen-restart/src/wen_restart.rs

```diff
@@ -0,0 +1,173 @@
+//! The `wen-restart` module handles automatic repair during a cluster restart
+
+use {
+    crate::solana::wen_restart_proto::{
+        MyLastVotedForkSlots, State as RestartState, WenRestartProgress,
+    },
+    log::*,
+    prost::Message,
+    solana_gossip::cluster_info::ClusterInfo,
+    solana_ledger::blockstore::Blockstore,
+    solana_vote_program::vote_state::VoteTransaction,
+    std::{
+        fs::File,
+        io::{Error, Write},
+        path::PathBuf,
+        sync::Arc,
+    },
+};
+
+// The number of ancestor slots sent is hard coded at 81000, because that's
+// 400ms * 81000 = 9 hours, we assume most restart decisions to be made in 9
+// hours.
+const MAX_SLOTS_ON_VOTED_FORKS: u64 = 81000;
```

Comment:
this constant should be just be `MAX_SLOTS_PER_ENTRY` I think based on this discussion:  https://github.com/solana-labs/solana/pull/33239#discussion_r1343082510

Reply:
Changed.